.github/actions/install-dependencies/action.yml

```diff
@@ -95,7 +95,7 @@ runs:
 
         brew install \
           ccache \
-          gmp \
+          texinfo \
```

Comment:
Is this required for building GMP? Can it be built without documentation?

Reply:
I removed the dependency. It turned out I had to fix a workaround we have in FindGMP when cross-compiling.